yogstation/code/datums/traits/neutral.dm

```diff
@@ -0,0 +1,76 @@
+/* Yogstation Neutral Traits */
+/datum/quirk/monochromatic
```

Comment:
this is already a thing though https://github.com/yogstation13/Yogstation-TG/blob/c24df24f82d1cdb444d2b18ffa43223e1fdcef89/code/datums/traits/neutral.dm#L70